src/js/angular/chatgpt/controllers.js

```diff
@@ -8,11 +8,14 @@ angular
     .controller('ChatGptCtrl', ChatGptCtrl)
     .controller('ChatGptSettingsCtrl', ChatGptSettingsCtrl);
 
-ChatGptCtrl.$inject = ['$scope', '$http', '$timeout', '$translate', '$uibModal', '$repositories', 'toastr', 'ModalService', 'LocalStorageAdapter'];
+ChatGptCtrl.$inject = ['$scope', '$http', '$timeout', '$translate', '$uibModal', '$repositories', 'toastr', 'ModalService', 'LocalStorageAdapter', '$licenseService'];
 
 const CHATGPTRETRIEVAL_ENDPOINT = 'rest/chat/retrieval';
 
-function ChatGptCtrl($scope, $http, $timeout, $translate, $uibModal, $repositories, toastr, ModalService, LocalStorageAdapter) {
+function ChatGptCtrl($scope, $http, $timeout, $translate, $uibModal, $repositories, toastr, ModalService, LocalStorageAdapter, $licenseService) {
+    $scope.isLicenseValid = function () {
```

Comment:
What's the reason to put this public function on top mixing it with the private once?

Reply:
Will move next to public